www/js/app.js

```diff
@@ -640,15 +640,64 @@ function focusPrefixOnHomeKey (event) {
         }, 0);
     }
 }
+
 /**
  * Verifies the given archive and switches contentInjectionMode accourdingly
- * @param {archive} the archive that needs verification
+ * @param {Object} archive The archive that needs verification
```

Comment:
Apologies, I just remembered that we have defined a type definition `{ZIMArchive}`, which my IDE recognizes, so perhaps best to use that here instead of `{Object}`.

Reply:
I actually did see that before, but I wasn't sure if it was the exact same class or not. I'll fix that